qt_gspeech_app:
  audio_rate: 16000
  default_language: 'en_US'